src/client/app/components/meters/CreateMeterModalComponent.tsx

```diff
@@ -544,25 +559,25 @@ export default function CreateMeterModalComponent(props: CreateMeterModalProps):
 					<Row xs='1' lg='2'>
 						{/* minVal input */}
 						<Col><FormGroup>
-							<Label for='minVal'>{translate('meter.minVal')}</Label>
+							<Label for='minVal'>{translate('minVal')}</Label>
 							<Input id='minVal' name='minVal' type='number'
 								onChange={e => handleNumberChange(e)}
 								min={MIN_VAL}
 								max={meterDetails.maxVal}
-								defaultValue={meterDetails.minVal}
+								value={meterDetails.minVal}
 								invalid={meterDetails?.minVal < MIN_VAL || meterDetails?.minVal > meterDetails?.maxVal} />
 							<FormFeedback>
 								<FormattedMessage id="error.bounds" values={{ min: MIN_VAL, max: meterDetails.maxVal }} />
 							</FormFeedback>
 						</FormGroup></Col>
 						{/* maxVal input */}
 						<Col><FormGroup>
-							<Label for='maxVal'>{translate('meter.maxVal')}</Label>
+							<Label for='maxVal'>{translate('maxVal')}</Label>
 							<Input id='maxVal' name='maxVal' type='number'
 								onChange={e => handleNumberChange(e)}
 								min={meterDetails.minVal}
 								max={MAX_VAL}
-								defaultValue={meterDetails.maxVal}
+								value={meterDetails.maxVal}
```

Comment:
Converting it to value is also not allowing to type a negative value. Removing a value in that field is setting the value to zero and then typing a negative value is impossible. Also please let me know whether that should be value or defaultValue in all the cases.

Reply:
I have not tried to implement this but my thinking is to look at src/client/app/components/admin/PreferencesComponent.tsx (newest development version) on setting the value with checks. Also, src/client/app/redux/selectors/adminSelectors.ts has the default values for meters. If a value was set for min/max rather than just the type I think the initial state would have the default value so it would work. Let me know if this makes sense/works or if I need to look at this more.